src/Symfony/Component/Console/Tests/Helper/HelperTest.php

```diff
@@ -15,6 +15,21 @@
 
 class HelperTest extends \PHPUnit_Framework_TestCase
 {
+    public function formatMemoryProvider()
+    {
+        return array(
+            array(0,          '0 B'),
```

Comment:
@phansys I formatted the array this way to be consistent with what's already in this file (take a look at the `formatTimeProvider` function).